FlexateWebApi.Application/Services/OfficesService.cs

```diff
@@ -94,13 +93,15 @@ public async Task<bool> UpdateOffice(int id, UpdateOfficeDto officeDto, Cancella
                 return false;
             }
 
-            Office office = new Office()
+            Office office = _mapper.Map<Office>(officeDto);
+            office.Id = id;
+
+            /*Office office = new Office()
```

Comment:
it's good practice to clean up any commented code, so it doesn't stay behind and clutter the project